easybuild/easyconfigs/f/FLAC/FLAC-1.3.1-foss-2015a.eb

```diff
@@ -0,0 +1,25 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+#
+# Author:    Stephane Thiell <sthiell@stanford.edu>
+###
+
+easyblock = 'ConfigureMake'
+
+name = 'FLAC'
+version = '1.3.1'
+
+homepage = 'https://xiph.org/flac/'
+description = """Programs and libraries for working with Free Lossless Audio Codec (FLAC) files."""
+
+toolchain = {'name': 'foss', 'version': '2015a'}
+
+sources = [SOURCELOWER_TAR_XZ]
+source_urls = ['http://downloads.xiph.org/releases/flac/']
+
+sanity_check_paths = {
+    'files': ['bin/flac', 'include/FLAC/all.h', 'lib/libFLAC.so'],
```

Comment:
please use `SHLIB_EXT` rather than hardcoding `.so`:

```python
    'files': ['bin/flac', 'include/FLAC/all.h', 'lib/libFLAC.%s' % SHLIB_EXT],
```